<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBook</class>
 <widget class="QDialog" name="AddBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter un nouveau livre</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/programme/images/icone.png</normaloff>:/programme/images/icone.png</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="labelTitre">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ajouter un nouveau livre</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignBottom">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Depuis Internet:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_recherche">
        <property name="text">
         <string>Titre, auteur, ISBN,...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_recherche">
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_recherche">
        <property name="text">
         <string>Go!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBoxLangue">
        <property name="text">
         <string>Rechercher sur tous les sites en langue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxLangue">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignBottom">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Manuellement:</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QPushButton" name="buttonManuel">
      <property name="text">
       <string>Ajouter manuellement un ouvrage</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_recherche</sender>
   <signal>clicked()</signal>
   <receiver>AddBook</receiver>
   <slot>addInternetBook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonManuel</sender>
   <signal>clicked()</signal>
   <receiver>AddBook</receiver>
   <slot>addManualBook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxLangue</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBoxLangue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addInternetBook()</slot>
  <slot>addManualBook()</slot>
 </slots>
</ui>
